omod/src/main/webapp/resources/htmlforms/mexico/consult.xml

```diff
@@ -430,20 +473,348 @@
+                        <lookup expression="fn.latestObs('CIEL:299').valueCoded.name"/>
+                    </p>
+                </div>
+                <div>
+                    <obs conceptId="CIEL:299"
+                         answerConceptIds="CIEL:1228,CIEL:1229" answerCodes="pihcore.reactive,pihcore.nonreactive"
+                         style="radio" answerSeparator="" />
+                </div>
+            </div>
+        </div>
+
+        <div class="question-container">
+            <label>
+                <uimessage code="pihcore.lab.hemoglobin"/>
+            </label>
+            <div class="two-columns">
+                <div class="small-text">
+                    <lookup complexExpression="#foreach($obs in $fn.allObs('CIEL:21')) $obs.valueNumeric — $obs.obsDatetime #end"/>
+                </div>
+                <div>
+                    <obs class="small" conceptId="CIEL:21" showUnits="true" />
+                </div>
+            </div>
+        </div>
+
+        <div class="question-container">
+            <label><uimessage code="pihcore.lab.urinary_albumin"/></label>
+            <div class="two-columns">
+                <div class="small-text">
+                    <p>
+                        <lookup expression="fn.latestObs('PIH:URINARY ALBUMIN').obsDatetime"/>
+                    </p>
+                    <p>
+                        <lookup expression="fn.latestObs('PIH:URINARY ALBUMIN').valueNumeric"/>
+                    </p>
+                </div>
+                <div>
+                    <obs class="small" conceptId="PIH:URINARY ALBUMIN" />
+                </div>
+            </div>
+        </div>
+
+        <div class="question-container">
+            <label for="blood-type">
+                <uimessage code="pihcore.lab.blood_type"/>
+            </label>
+            <div class="two-columns">
+                <div class="small-text">
+                    <lookup expression="fn.latestObs('PIH:BLOOD TYPING').valueCoded.name"/>
+                </div>
+                <div>
+                    <p class="radio" >
+                        <obs conceptId="PIH:BLOOD TYPING"
+                             answerConceptIds="CIEL:690,CIEL:692,CIEL:694,CIEL:696,CIEL:699,CIEL:701,CIEL:1230,CIEL:1231"
+                             answerCodes="A+,A-,B+,B-,O+,O-,AB+,AB-" style="radio"/>
+                    </p>
+                </div>
+            </div>
+        </div>
+
+        <div class="question-container">
+            <label for="blood-type">
+                <uimessage code="pihcore.mch.dtpGiven"/>
+            </label>
+            <div class="two-columns">
+                <div class="small-text">
+                    <lookup complexExpression="#foreach($obs in $fn.allObs('CIEL:984')) $obs.valueCoded.getShortNames() — $obs.obsDatetime #end"/>
+                </div>
+                <div>
+                    <obs conceptId="CIEL:984" answerConceptId="CIEL:781" answerCode="yes"/>
```

Comment:
Ended up modeling this using https://concepts.pih-emr.org/openmrs/dictionary/concept.htm?conceptId=10156